### ISSUE TITLE
Reconsider WOFF2

### ISSUE THREAD

**Issue by mathiasbynens** [mmastrac/webfont-dl#8]
https://github.com/mmastrac/webfont-dl#woff-v2

Now that WOFF2 is supported by Chrome, Opera, and Firefox, has the time come to download WOFF2 font versions by default?

**Comment by mmastrac** [comment]
Still waiting on Safari and for WOFF2 to make it into the stable channels of every browser unfortunately. I would wager within a year this can flip. Leaving open as a tracking bug.

**Comment by konklone** [comment]
@mmastrac If WOFF2 were listed as an option in the resulting `.css` file, would browsers which don't support it try and fail to use it, and have broken fonts? Is there any downside to including WOFF2 in the file for browsers that don't support it?

**Comment by mmastrac** [comment]
Oh crap -- I think I just realized the confusion. The WOFF2 support page is here: http://caniuse.com/#feat=woff2 (I linked to WOFF by accident)

I'll fix the link on the front page. Does this make sense?

**Comment by mathiasbynens** [comment]
@konklone 

> If WOFF2 were listed as an option in the resulting `.css` file, would browsers which don't support it try and fail to use it, and have broken fonts?

No.

> Is there any downside to including WOFF2 in the file for browsers that don't support it?

No.

TL;DR Everything is awesome!

**Comment by mmastrac** [comment]
The downside is basically size, which is somewhat antithetical to the reasons you'd use WOFF2: the byte savings. 

If you're including WOFF+WOFF2, that'll be on average ~170% of the size of a WOFF-only CSS file.

If you inline WOFF2 and link WOFF, cutting edge browsers will potentially save 30-50% on the CSS download, but modern browsers that don't speak WOFF2 yet (Safari, IE) will pay a reasonably large penalty having downloaded the CSS, ignored the WOFF2 and moved on to WOFF.

I'd say that if you can guarantee that most of your users are on Chrome and Firefox 39+, then enabling WOFF2 is probably a win. If not, I'd suggest sticking with the default.

**Comment by mmastrac** [comment]
I'm quickly revisiting this since the web is moving quickly these days -- http://caniuse.com/#feat=woff2 suggests that WOFF2 is 50%+ of the global market share which means that it might be worth adding a non-default flag to enable this profile.

Adding help wanted.

**Comment by mmastrac** [comment]
woff2 support is now at 80% - when I get a chance I'll make this the default.

**Comment by mmastrac** [comment]
Committed
